experiments/openebs/openebs-pool-network-loss/openebs_pool_network_loss_ansible_logic.yml

```diff
@@ -0,0 +1,174 @@
+- hosts: localhost
+  connection: local
+
+  vars:
+    a_label: "{{ lookup('env','APP_LABEL') }}"
+    a_ns: "{{ lookup('env','APP_NAMESPACE') }}"
+    a_pvc: "{{ lookup('env','APP_PVC') }}"
+    c_duration: "{{ lookup('env','TOTAL_CHAOS_DURATION') }}"
+    c_experiment: "openebs-pool-network-loss"
+    c_interval: "5"
+    data_persistence: "{{ lookup('env','DATA_PERSISTENCE') }}"
+    lib_image: "{{ lookup('env','LIB_IMAGE') }}"
+    liveness_label: "{{ lookup('env','LIVENESS_APP_LABEL') }}"
+    liveness_namespace: "{{ lookup('env','LIVENESS_APP_NAMESPACE') }}"
+    packet_loss_perc: "{{ lookup('env','NETWORK_PACKET_LOSS_PERCENTAGE') }}"  
+    openebs_ns: "{{ lookup('env','OPENEBS_NAMESPACE') }}"
+
+  vars_files:
+    - /mnt/parameters.yml
+    - /experiments/openebs/openebs_components.yml
+  
+  tasks:
+    - block:
+      - include_tasks: /utils/runtime/create_testname.yml  
+
+      - name: Identify the chaos util to be invoked 
```

Comment:
Can you please add this step in openebs_pool_network_loss_ansible_prerequisites.yml file?